00-programming-fundamentals/exercises/regex-exercises.md

```diff
@@ -0,0 +1,33 @@
+
+# Regex Practice
```

Comment:
There are really good (and will be really challenging haha). I personally would really want your solutions in the TET for this.

Reply:
I'll add them to the TET today.